### ISSUE TITLE
Linking to specific queries?

### ISSUE THREAD

**Issue by m-mohr** [radiantearth/stac-api-spec#420]
We'd like to link to ItemCollections that are search results, so URLs like:

https://example.com/api/search?filter=collection%20%3D%20%27landsat8_l1tp%27%20AND%20eo%3Acloud_cover%20%3C%3D%2010

or 

https://example.com/api/collections/landsat8_l1tp/items?filter=eo%3Acloud_cover%20%3C%3D%2010

I'd like that clients read them similarly to how the Items are consumed. What's the best relation type for it? Can "items" be used? Or something new like "search-result"?

**Comment by gadomski** [comment]
I prefer having fewer things. Reusing `items`+`data`, instead of adding new rel types to our ecosystem, seems better to me. Can you lay out the reasons why `items`+`data` _couldn't_ be reused?

**Comment by gadomski** [comment]
> if you want to have the "original" and specific queries side by side, how to distinguish them?

Yup, agreed -- I know I've written a client that looks for `data` to get the collections link, for example. So yeah, that changes my answer -- new rel type seems better :-P. Thanks!

As for naming, yours seem good enough, if a bit long. I didn't see anything about "search-result" in the IANA (I went looking hoping for a good analogue and came up blank).
